pkg/xds/bootstrap/server_test.go

```diff
@@ -62,7 +62,7 @@ var _ = Describe("Bootstrap Server", func() {
 	}`
 
 	ca := x509.NewCertPool()
-	cert, err := ioutil.ReadFile(filepath.Join("..", "..", "..", "test", "certs", "server-cert.pem"))
+	cert, err := os.ReadFile(filepath.Join("..", "..", "..", "test", "certs", "server-cert.pem"))
```

Comment:
It might worth adding a `gomodguard` rule to exclude `io/ioutil` to replace it with `os` or `io`